requirements/extras/test_requirements.txt

```diff
@@ -0,0 +1,20 @@
+tox==3.24.5
+flake8==4.0.1
+pytest==6.0.2
+pytest-cov==3.0.0
+pytest-rerunfailures==10.2
+pytest-timeout==2.1.0
+pytest-xdist==2.4.0
+coverage>=5.2, <6.2
```

Comment:
mismatch for coverage version here and in other 2 files

Reply:
Thanks Will make this consistent.